client/src/components/Members.js

```diff
@@ -1,10 +1,24 @@
 import React, { Component } from 'react';
 import '../index.css';
+import MemberList from './MemberList'
+import Select from 'react-select'
 
 // Search page for members
 class Members extends Component {
   constructor(props) {
     super(props);
+    const genderOptions = [
+      { value: 'Male', label: 'Male' },
+      { value: 'Female', label: 'Female' },
+      { value: 'Other', label: 'Other' }
+    ]
+    const statusOptions = [
+      { value: 'Single', label: 'Single' },
+      { value: 'Couple', label: 'Couple' },
+      { value: 'ParentChild', label: 'Couple with Children' },
+      { value: 'SingleParent', label: 'Single Parent' },
+      { value: 'Group', label: 'Existing Group' }
+    ]
```

Comment:
Just a note: in the future, these options will be generated from our database tables which hold this data. But, for now, it makes sense to fill these options manually.